src/functions/getJournal/application/service/getDynamicJournal.ts

```diff
@@ -0,0 +1,406 @@
+import { ExaminerWorkSchedule } from '../../../../common/domain/Journal';
+import * as logger from '../../../../common/application/utils/logger';
+
+export function getDynamicJournal(): ExaminerWorkSchedule {
+  logger.info('Returning static journal...');
```

Comment:
Logging message seems at odds with the function name - very trivial i know.

Reply:
Thanks, sorted.